src/test/java/io/confluent/connect/hdfs/avro/DataWriterAvroTest.java

```diff
@@ -326,7 +331,8 @@ public void testRebalance() throws Exception {
     }
 
     hdfsWriter.write(sinkRecords);
-    hdfsWriter.close();
+    hdfsWriter.close(assignment);
```

Comment:
I think you are referring to the close in testRebalance. In this case, assignment is already refers to the NewAssignment.